docs/contributing/stub-list.md

```diff
@@ -1,10 +0,0 @@
----
```

Comment:
If that was deleted, were all stubs references removed from the docs?

Reply:
Good point. I just deleted and set up redirects for all the ones except the ones under "using gatsby professionally". (There's a different story to figure out what to do with those ones, since they should probably be pulled out of the docs site.)